### PR TITLE
Update vaccine Q&A for severe covid chance

### LINKED ISSUE #832: Vaccine Q&A summary at odds with the rest of the section
Hi, I'm confused by this in the vaccines section of the Q&A (emphasis mine):

> Summary: Vaccinated people receive a multiplier on all incoming microCOVIDs from their activities. The multiplier depends on which vaccine they've gotten and is contingent on having waited long enough for their immune system to respond. **Additionally, each microCOVID may be 2-4x less likely to land a vaccinated individual in a hospital.**

vs.

> Data from Israel's study suggests that Pfizer's vaccine reduced cases of severe COVID by 92%, about the same as the reduction in mild/asymptomatic COVID. **Therefore we do not think that mRNA vaccines reduce the chance of hospitalization or death beyond the reduction in contracting COVID in the first place.** It is possible that the traditional vaccines (AstraZenica, Johnson&Johnson) are somewhat more protective againt severe COVID than mild COVID, but we have not seen compelling data for this.
> [...]
> As a result, getting a vaccination yourself confers a reduction in incoming microCOVIDs (as outlined above), but **we do not recommend increasing your microCOVID budget on top of that given current data.**

The "2-4x less likely" claim in the summary is never addressed and is indeed contradicted in the section it says it's summarizing, such that this makes me think this may be an editing error or a disagreement of some sort? Could this be clarified?

(Related side note - it was definitely my impression from just seeing the vaccine multipliers in the calculator and comparing them to the headline vaccine efficacy numbers I'd seen elsewhere that the assumption here is that the vaccines are less effective at preventing getting covid at all than they are at preventing people from _actually getting sick_, so I was and still kind of am somewhat confused by the combination of the rather conservative vaccine multipliers + the claim that vaccines are not more protective against worse outcomes than against getting covid at all (since you'd think that if you're less likely to have *any symptoms* conditional on having covid at all, you'd also be less likely to have *worse* symptoms). This specific confusion might be worth addressing in the Q&A and/or the research sources, though I realize this is a big topic and possibly best addressed elsewhere.)

### PR DESCRIPTION
**Pull request by beshaya**
Fixes #832

### CODE REVIEW

**Comment by blanchardjeremy** [comment]
@beshaya  I don't think I'm best to review this since im not really up to date on the technical stuff. Unless you just want a grammatical check. Let me know.

File: src/posts/paper/13-q-and-a.ts [hunk 1]
```diff
@@ -212,16 +212,26 @@ While the vaccines are nowhere near making you invulnerable to COVID, remember t
 
 ### 2. If a vaccinated individual contracts COVID, how much less (or more) likely is this to result in negative consequences? (Increased budget)
 
-Data from Israel's study suggests that Pfizer's vaccine reduced cases of severe COVID by 92%, about the same as the reduction in mild/asymptomatic COVID. Therefore we do not think that mRNA vaccines reduce the chance of hospitalization or death beyond the reduction in contracting COVID in the first place. It is possible that the traditional vaccines (AstraZenica, Johnson&Johnson) are somewhat more protective againt severe COVID than mild COVID, but we have not seen compelling data for this.
+Data from Israel's study suggests that Pfizer's vaccine reduced cases of severe COVID by 92% (95% CI, 75 to 100), about the same as the reduction in mild/asymptomatic COVID.
+Therefore we do not think that mRNA vaccines significantly reduce the chance of hospitalization or death beyond the reduction in contracting COVID in the first
+place. Note that the error bars on this figure are somewhat wide; the risk reduction is likely somewhere between 3x and 1/3x.
```
[the remark this answers is not in the document]
Comment (beshaya) [reply src/posts/paper/13-q-and-a.ts]: Outdated.

File: src/posts/paper/13-q-and-a.ts [hunk 2]
```diff
@@ -212,16 +212,26 @@ While the vaccines are nowhere near making you invulnerable to COVID, remember t
 
 ### 2. If a vaccinated individual contracts COVID, how much less (or more) likely is this to result in negative consequences? (Increased budget)
 
-Data from Israel's study suggests that Pfizer's vaccine reduced cases of severe COVID by 92%, about the same as the reduction in mild/asymptomatic COVID. Therefore we do not think that mRNA vaccines reduce the chance of hospitalization or death beyond the reduction in contracting COVID in the first place. It is possible that the traditional vaccines (AstraZenica, Johnson&Johnson) are somewhat more protective againt severe COVID than mild COVID, but we have not seen compelling data for this.
+Data from Israel's study suggests that Pfizer's vaccine reduced cases of severe COVID by 92% (95% CI, 75 to 100), about the same as the reduction in mild/asymptomatic COVID.
+Therefore we do not think that mRNA vaccines significantly reduce the chance of hospitalization or death beyond the reduction in contracting COVID in the first
```
[the remark this answers is not in the document]
Comment (beshaya) [reply src/posts/paper/13-q-and-a.ts]: Changed.

File: src/posts/paper/13-q-and-a.ts [hunk 3]
```diff
@@ -212,16 +212,26 @@ While the vaccines are nowhere near making you invulnerable to COVID, remember t
...
+As a result, getting a vaccination yourself confers a reduction in incoming microCOVIDs (as outlined [above](#1-how-much-less-likely-is-it-for-vaccinated-individuals-to-catch-covid-reduction-in-microcovids)).
+but we do not have a general recommendation on changing your microCOVID budget after vaccination.
+
+Some reasons you might want to change your budget:
+* If you live with housemates, you can increase your personal budget while maintaining the same annual risk; each housemate's activites has a reduced impact on others' microCOVID budget. I.e. if living with N housemates, before vaccination each person's allocation of microCOVIDs would have been (200 / (1 + 0.3N)). After everyone is vaccinated, this increases to (200 / (1 + 0.03N)).
+* You may have been choosing your microCOVID budget primarily based on mitigating risk to others, rather than concern for your own health; When both you and the people you are trying to protect are vaccinated, you may increase your budget by 1 / vaccine multiplier (i.e. 10x for the mRNA vaccines) while still having the same impact on them. 
+
```
[the remark this answers is not in the document]
Comment (beshaya) [reply src/posts/paper/13-q-and-a.ts:231]: Mmm, no. The first point says "this change will maintain 200 uCoV for yourself", the second says "have the same impact on others while going wild"

For a house of 6, you'd originally be at 80. Following the first you could go to 173. Following the second you could go to 800.

Another way to think is, the second is what it's fair to ask of your housemates if you are risk averse. The first is what you might want to do yourself if you are risk averse.

[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/posts/paper/13-q-and-a.ts:231]: I wonder how we might make this section more concrete with a full example. Basically saying what you just said. That makes it much more clear to me.

File: src/posts/paper/13-q-and-a.ts [hunk 4]
```diff
@@ -212,16 +212,37 @@ While the vaccines are nowhere near making you invulnerable to COVID, remember t
...
 
-If you do decide you want to increase your microCOVID budget post vaccination,
-* Discuss this with any regular close contacts you have, such as housemates, partners, or co-workers — your increased risk of COVID, asymptomatic or not, could affect them more than it affects you.
-* Consider the potential impact on your community at large — taking on more risk of being an asymptomatic spreader could result in you infecting many other people.
+As a result, getting a vaccination yourself confers a reduction in incoming microCOVIDs based on how the vaccine affects your chances of infecting others (as outlined [above](#1-how-much-less-likely-is-it-for-vaccinated-individuals-to-catch-covid-reduction-in-microcovids)).
+You may additionally increase your budget based on how the vaccine reduces your own chances of negative outcomes, but this is a personal choice dependent on your own view of how averse to the various possible negative outcomes you are.
+
+Some ways you may want to change your budget in response to getting vaccinated people around you vaccinated:
```
Comment (zerotrickpony) [inline src/posts/paper/13-q-and-a.ts:230]: Just a small clarity comment, apologies if this isn't helpful: "getting vaccinated people around you vaccinated" on line 230 seems redundant/unclear? Did we mean to say:

`"...in response to people around you getting vaccinated"?`

Comment (beshaya) [reply src/posts/paper/13-q-and-a.ts]: Thanks for catching that typo!

File: src/posts/paper/13-q-and-a.ts [hunk 5]
```diff
@@ -212,16 +212,37 @@ While the vaccines are nowhere near making you invulnerable to COVID, remember t
...
+
+You live in a pod of 6. Prior to vaccination, all 6 housemates had a personal budget of \`200 / (1 + .3 * 5) = 80 microCOVID/week\`. All 6 of you are now vaccinated.
+* If all 6 of you agree you want to keep to a 200 microCOVID budget, you adjust the formula to \`200 / (1 + .03 * 5) = 174 microCOVID/week\`.
+* If you agree to keeping the same impact on each other while individually choosing your own risks, you each may go up to \`80 / 0.1 = 800 microCOVID/week\`.
+* If you all decide that you want to keep the same risk of getting severe COVID as pre-vaccination, you further double your budget (accepting a higher risk of mild/asymptomatic COVID).
+
+Example B:
+
+You live with or care for someone with a condition that reduces the efficacy of a vaccine (consult their doctor).
+* You choose not to change your budget, since this person is just as likely to get COVID from you as before.
```
Comment (blanchardjeremy) [inline src/posts/paper/13-q-and-a.ts:245]: love this. thank you!

**Review by blanchardjeremy — APPROVED** [review]
Gave it a quick skim. LGTM.

File: src/posts/paper/13-q-and-a.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (jfslocum) [reply src/posts/paper/13-q-and-a.ts:234]: > When both you and the people you are trying to protect are vaccinated, you may increase your budget by 1 / vaccine multiplier (i.e. 10x for the mRNA vaccines) while still having the same impact on them. 

Your vaccine doesn't actually matter here: it's just theirs that matters.

[the remark this answers is not in the document]
Comment (beshaya) [reply src/posts/paper/13-q-and-a.ts:234]: You are correct. This PR has merged, could you please open a new PR to fix that?

**Pull request merged**